app/services/notifications/setup/batch-notifications.service.js

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+/**
+ * Orchestrates sending notifications to notify and saving the notification to 'water.scheduled_notifications'
+ * @module BatchNotificationsService
+ */
+
+const CREATED = 201
+
+const NotifyEmailService = require('../../notify/notify-email.service.js')
+const NotifyLetterService = require('../../notify/notify-letter.service.js')
+const ReturnsNotificationPresenter = require('../../../presenters/notifications/setup/returns-notification.presenter.js')
+
+/**
+ * Orchestrates sending notifications to notify and saving the notification to 'water.scheduled_notifications'
+ *
+ * This service needs to perform the following actions on a recipient:
+ * - convert the recipients into notifications
+ * - send the notification to notify
+ * - save the notifications to 'water.scheduled_notifications'
+ * - check the status of the notifications with notify (sent or otherwise)
+ * - update the notifications status in 'water.scheduled_notifications' (with the updated notify status)
+ * - return the amount of successful and failed notifications
+ *
+ * This will need to be done in batches because Notify has a rate limit (3,000 messages per minute), and the persisting of the
+ * 'water.scheduled_notifications' uses postgresql's ability to insert with batches (this is much grater than the Notify
+ * rate limit).
```

Comment:
```suggestion
 * This service needs to perform the following actions on a recipient:
 
 * - convert the recipients into notifications
 * - send the notifications to Notify
 * - save the notifications to `water.scheduled_notifications`
 * - check the status of the notifications with Notify (sent or otherwise)
 * - update the notifications status in `water.scheduled_notifications` (with the updated Notify status)
 * - return the number of successful and failed notifications
 *
 * This will need to be done in batches because Notify has a rate limit (3,000 messages per minute). It also means the persisting of the notifications can use PostgreSQL's ability to batch insert (this is much greater than the Notify
 * rate limit).
```